WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -20,6 +20,38 @@ class CommentDetailViewController: UITableViewController {
         return navigationController?.visibleViewController == self
     }
 
+    private var replyID: Int32 {
+        didSet {
+            // toggle reply indicator cell visibility as needed.
+            // to prevent the content cell from reloading its web content, only the specific index path is targeted.
+            switch (oldValue, replyID) {
+            case (.zero, 1...Int32.max):
```

Comment:
I find these `switch` `case` statement confusing and hard to read. I figured it out after thinking about it, but honestly IMO I shouldn't have to think that hard about what each `case` does. 😄 Can these be clearer / more readable?

Reply:
Thanks for the input! I've tried to improve this in https://github.com/wordpress-mobile/WordPress-iOS/pull/17409/commits/bfb2b792e0d7b62fcdc579c8f6e7a108594865cc by using the traditional if/else. Hope this is a bit clearer now or otherwise please let me know! 🙂

Reply:
Thank you!!